python-sdk/actionml/connection.py

```diff
@@ -203,6 +206,7 @@ def set_request(self, request):
 
 class ActionMLHttpConnection(object):
```

Comment:
the project name is Harness, not ActionML. The package is misnamed and should be `harness`